### DISCUSSION TITLE
Some configuration help needed

### DISCUSSION THREAD

**Discussion by bound-variable** [baptistecdr/aria2-extensions#27] [Q&A] [answered]
I got **aria2** to work as a daemon with [another Chrome extension](https://chrome.google.com/webstore/detail/aria2c-integration-2/nhkaifgoebiidfpmdhmbindnfkpglnem/related), but not this one. The problem is this extension doesn't register any running server.

My service file:
```
[Unit]
Description=Aria2 Daemon
After=network.target

[Service]
Type=forking
ExecStart=/usr/bin/aria2c --conf-path=/etc/aria2.daemon

[Install]
WantedBy=multi-user.target
```

My daemon file:
```
continue=true
daemon=true
dir=/home/alan/Downloads/
file-allocation=falloc
log-level=warn
max-connection-per-server=1
max-concurrent-downloads=1
enable-rpc=true
rpc-listen-all=true
rpc-allow-origin-all=true
#rpc-user=...
#rpc-passwd=...
```

Finally, my error:
![image](https://user-images.githubusercontent.com/51193876/142781934-d713018a-bdc9-4940-b4f4-55e4445098f1.png)

Any advice?

Thanks

- Debian 11
- Gnome 3.38.5
- Aria2 1.35.0-3
- Aria2 Integration 3.2.1
- Browser: Brave 1.32.106

**Comment by baptistecdr** [comment, marked as answer]
Hi Alan,

You need to click on the "+" sign to create a new configuration server.
Then you need to fill the form: name, host, password, http/s, etc.

> https://github.com/baptistecdr/aria2-extensions/blob/master/img/firefox/options_server_light.png

After that, in "Options" tab, you can select your server in "Server to send downloads".

Let me know if you have any other trouble.

**Reply by nollinvoyd** [reply]
HI,
Where do I go to find this info: name, host, password, http/s, etc.

The image from your last post is no longer available.

![aria - Copy](https://github.com/user-attachments/assets/630065de-d6fd-41dc-a9d6-d67ac8dc17c1)

thanks

**Reply by baptistecdr** [reply]
Hello @nollinvoyd,

For the name, you can choose whatever you want.
For:
- host, if you installed Aria2 locally, it will be localhost, otherwise the IP/domain of the machine
- port, by default, it is on port 6800
- secure connection, unchecked if you didn't setup any SSL certificate
- Secret, the value that you put on the RPC parameter (--rpc-secret) in Aria2. Leave empty if you didn't put anything.

Let me know if that works for you. :)

**Reply by nollinvoyd** [reply]
Hi @baptistecdr 

Here's what i have now.  I gave it a name, and added my IP address.

![aria3](https://github.com/user-attachments/assets/79baa9d9-6066-4380-8a3a-1b05e7f07297)

Can I assume it will not work on YouTube? Doesn't for me.  Can you provide a a link to somewhere I can test it?

I use Cloudfare.  Is that a factor?  I don't know what to do about an ssl certificate

**Comment by bound-variable** [comment]
Ah yes, the plus sign. I saw that and thought it opened a new tab. I thought I was looking at the first tab housing the first configuration. 🤪

Cheers

Sent from ProtonMail mobile

-------- Original Message --------
